app/src/main/java/chat/rocket/android/helper/Constants.kt

```diff
@@ -0,0 +1,5 @@
+package chat.rocket.android.helper
+
+object Constants {
```

Comment:
@filipedelimabrito I don't think I know how to handle this, can you provide some guidance please?  This const is used in multiple classes, across multiple activities.  Should we be declaring it uniquely in each class?  Or is there a more general way to do it?  Thanks!

Reply:
Let me take a look in a deep way here. But since we are going to use a Constant class with, well... our constants we will need to move all the stuffs declared on our code here to avoid people declaring it in multiple places of our code for instance.